.github/workflows/release.yml

```diff
@@ -0,0 +1,119 @@
+# must do before using this action:
+# - set up RELEASE_TOKEN in secrets of the repository avocado-framework/avocado
+# - set up RTD_TOKEN in secrets to update readthedocs
+# - set up two tokens for twine: PYPI_USER and PYPI_PASSWD
+
+name: Release tests
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version'
+        required: true
+        default: '0.0'
+      devel_name:
+        description: 'Developer Name'
+        required: true
+        default: 'Avocado Developer'
+      devel_mail:
+        description: 'Developer mail'
+        required: true
+        default: 'avocado@redhat.com'
+      rtd_project:
+        description: 'readthedocs project name'
+        required: true
+        default: 'avocado-framework'
+
+jobs:
+
+  release:
+    name: Release pipeline
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:34
+
+    steps:
+      - name: install required packages
+        run:  dnf -y install rpmdevtools git python3-pip make
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - name: Update VERSION files and python-avocado.spec
+        run: |
+          export VERSION=${{ github.event.inputs.version }}
+          export DEVEL_NAME=${{ github.event.inputs.devel_name }}
+          export DEVEL_MAIL=${{ github.event.inputs.devel_mail }}
```

Comment:
Hi @ana I'm trying to understand why those exports are needed here (here and in other parts of this file).

I did some tests using the `env` section and exporting directly the variables. It seems to be working, including for multiple steps:


```yaml
jobs:

  release:
    name: Release pipeline
    runs-on: ubuntu-latest
    container:
      image: fedora:34
    env:
      DEVEL_NAME: ${{ github.event.inputs.devel_name }}
      DEVEL_MAIL: ${{ github.event.inputs.devel_mail }}

    steps:
      - name: install required packages
        run:  dnf -y install rpmdevtools git python3-pip make
      - uses: actions/checkout@v2
        with:
          persist-credentials: false
          fetch-depth: 0
      - name: Update VERSION files and python-avocado.spec
        run: |
          set | grep DEVEL
          make test
      - name: Commit files and tag 
        run: |
          set
          make test
```

This example is here: https://github.com/beraldoleal/foobartest/

What I'm missing?

Reply:
@beraldoleal I had the feeling that could be changed in the way you mentioned, but I didn't have time to test properly, I focused in checking the secret tokens weren't showed in the github action logs.
I found another problem with the variables so I'll include this change as well.